docs/rules/consistent-this.md

```diff
@@ -23,6 +23,12 @@ This rule designates a variable as the chosen alias for `this`. It then enforces
 
 This rule takes one option, a string, which is the designated `this` variable. The default is `that`.
 
+Additionally, you may configure extra aliases for cases where there are more than one supported alias for `this`.
+
+```js
+{ "consistent-this": [ 2, "self", { "extraAllowedAliases": [ "vm" ] } ] }
```

Comment:
I would prefer below:

```json
{
    "consistent-this": [2, "self", "vm"]
}
```

or

```json
{
    "consistent-this": [2, ["self", "vm"]]
}
```

--------

JSON schema would be:

The former's:

```js
module.exports.schema = {
    "type": "array",
    "items": {
        "type": "string",
        "minLength": 1
    },
    "minItems": 1,
    "uniqueItems": true
};
```

The latter's:

```js
module.exports.schema = [
    {
        "oneOf": [
            {
                "type": "string",
                "minLength": 1
            },
            {
                "type": "array",
                "items": {
                    "type": "string",
                    "minLength": 1
                },
                "minItems": 1,
                "uniqueItems": true
            }
        ]
    }
];
```